README.md

```diff
@@ -3,6 +3,8 @@
 
  # e-commerce-ones-and-zeroes-bn
 Ones-and-Zeroes e-commerce backend
+POSTGRESS SEQUILIZER
 
+Description
```

Comment:
Prefer providing us a descriptive Readme, if you want. Or else remove unnecessary details